src/components/NewAppModal.js

```diff
@@ -130,8 +129,12 @@ export const NewAppModal = _.flow(
             ]),
             li({ style: { marginTop: '1rem' } }, [
               'Running cloud compute costs ',
-              span({ style: { fontWeight: 600 } }, '$0.00 per hr')
-              //TODO: Calculate cost
+              span({ style: { fontWeight: 600 } }, `${Utils.formatUSD(
+                hourlyAppCost(
+                  app ? app.kubernetesRuntimeConfig : { machineType: 'n1-standard-8' },
+                  app ? app.status : 'RUNNING'
+                ) + persistentDiskCost({ size: 30, status: 'Running' })
```

Comment:
I think in our other PR we changed the default disk size to 50

Reply:
I thought we observed that the default for Galaxy was 30 (coming from Leo)

Reply:
It is, there was some confusion cause the text still says 50 in the launch PR, I think, so commented there instead.